modules/mono/glue/GodotSharp/GodotSharp/Core/Transform3D.cs

```diff
@@ -175,11 +175,14 @@ readonly get
         /// </summary>
         /// <param name="target">The object to look at.</param>
         /// <param name="up">The relative up direction.</param>
+        /// <param name="useModelFront">If specified, then the model is oriented in reverse,
+        /// towards the model front axis (+Z, Vector3.MODEL_FRONT),
+        /// which is more useful for orienting 3D models.</param>
         /// <returns>The resulting transform.</returns>
-        public readonly Transform3D LookingAt(Vector3 target, Vector3 up)
+        public readonly Transform3D LookingAt(Vector3 target, Vector3 up, bool useModelFront = false)
```

Comment:
Same as the method from `Basis`, this breaks binary compatibility and the method in Core also seems to have a default value for the `up` parameter.

```csharp
public Transform3D LookingAt(Vector3 target, Nullable<Vector3> up = null, bool useModelFront = false)
{
	up = up.HasValue ? up.Value : new Vector3(0, 1, 0);
```